edx-platform/pearson-theme/lms/templates/index.html

```diff
@@ -85,3 +111,56 @@ <h1 class="title">
   $(window).load(function() {$('#signup_action').trigger("click");});
 </script>
 % endif
+
+%if search_courses_home:
+  <script type="text/javascript">
+    var searchHome = (function() {
+      'use strict';
+
+      var inputField, coursesList, filters, iconClear, valueInput;
+
+      function initElements() {
+        inputField = $("#search-input");
+        coursesList = $(".courses-listing li");
+        filters = $(".quick-filter-item");
+        iconClear = $(".clear-search-icon");
+      }
+
+      function eventSearch() {
+        inputField.on("input", function(){
+          valueInput = $(this).val().toLowerCase();
+          coursesList.filter(function(){
+           $(this).toggle($(this).text().toLowerCase().indexOf(valueInput) > -1)
+          });
+        });
+      } 
+
+      function eventFilters() {
+        filters.on("click", function(){
+          inputField.val($(this).text()).trigger("input");  
+        });
+      }
+
+      function clearSearch() {
+        iconClear.on("click", function() {
+          inputField.val('').trigger("input");
```

Comment:
If you are going to use single quotes or double quotes, be sure to use it it throughout Javascript block. @amalbas